test/config/ConfigBuilderTest.js

```diff
@@ -19,6 +19,7 @@ var path = require('path');
 var ConfigBuilder = require('../../lib/config/ConfigBuilder').ConfigBuilder;
 var Config = require('../../lib/index').Config;
 var Long = require('long');
+const createAddressFromString = require("../../lib/Util").createAddressFromString;
```

Comment:
Minor comment: `const` is not fully supported in Node.js 4. It does not apply the semantics that it implies. So I do not recommend using it in tests (In actual source code, it is different story because TS compiler actually enforces const semantics). It would not cause any problem here so it is just a style issue.